src/s2/s2polyline.cc

```diff
@@ -818,6 +818,8 @@ void S2Polyline::Shape::Init(const S2Polyline* polyline) {
   polyline_ = polyline;
 }
 
+S2Polyline::OwningShape::~OwningShape() = default;
+
```

Comment:
Since `OwningShape` is declared after `Shape`, it seems like it would make sense to define this at the bottom, after the `Shape` definitions.

Reply:
Sure, can do.